api/src/main/java/io/strimzi/api/kafka/model/KafkaResources.java

```diff
@@ -71,6 +71,17 @@ public static String entityOperatorSecretName(String clusterName) {
         return entityOperatorDeploymentName(clusterName) + "-certs";
     }
 
+    /**
+     * Returns the name of the Cruise Control {@code Deployment} for a {@code Kafka} cluster of the given name.
+     * This {@code Deployment} will only exist if {@code Kafka.spec.cruiseControl} is configured in the
+     * {@code Kafka} resource with the given name.
+     * @param clusterName  The {@code metadata.name} of the {@code Kafka} resource.
+     * @return The name of the corresponding Entity Operator {@code Deployment}.
+     */
+    public static String cruiseControlDeploymentName(String clusterName) {
+        return clusterName + "-cruise-control";
+    }
+
```

Comment:
CC deployment name can be grabbed from here[1]

[1] https://github.com/strimzi/strimzi-kafka-operator/blob/7ad2d67802debaa9d8c81df1b56635add99c1a5c/api/src/main/java/io/strimzi/api/kafka/model/CruiseControlResources.java#L18-L20